webapp/next.config.js

```diff
@@ -1,3 +1,8 @@
+// @typescript-eslint expects usage of imports
+/* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
Perhaps this file should be declared as CommonJS in the linter/TS configs.

Reply:
Fixed here [5d1732e](https://github.com/hemilabs/ui-monorepo/pull/622/commits/5d1732e110652db4db02bdb06cfaeba6db30fcbe)